py2puml/py2puml.py

```diff
@@ -1,7 +1,7 @@
 from typing import Dict, List, Iterable
 from types import ModuleType
-from pkgutil import walk_packages
-from importlib import import_module
+import pkgutil
+import importlib
```

Comment:
please, do not use global imports. Can you revert these changes and follow the approach used throughout the codebase of this library, please? For consistency sake.

Reply:
Ok I will. This change added for mocking easily in the unit test, but now unit test case have no mock object.